sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -488,7 +492,9 @@ def plan_to_named_target(self, name, custom_start_state=None):
         if self.set_named_target(name):
             self.__plan = self._move_group_commander.plan()[CONST_TUPLE_TRAJECTORY_INDEX]
         else:
-            rospy.logwarn("Failed to set to named target")
+            rospy.logwarn("Could not find named target, plan not generated")
+            return False
+        return True
```

Comment:
with current method name I'd expect a plan to be returned

Reply:
mmm or just whether it has planned or not. it's not get_plan_ , need to check, it's not great to return a private variable (where we are currently storing the plan)